alluxio-ui/worker/src/containers/pages/Metrics/Metrics.tsx

```diff
@@ -119,10 +116,11 @@ class Metrics extends React.Component<AllProps> {
   }
 }
 
-const mapStateToProps = ({metrics}: IApplicationState) => ({
+const mapStateToProps = ({metrics, refresh}: IApplicationState) => ({
   errors: metrics.errors,
   loading: metrics.loading,
-  metrics: metrics.metrics
+  metrics: metrics.metrics,
```

Comment:
We have a member variable `metrics` and a local variable `metrics`, how does this work?

Reply:
Yeah, I started with metrics simply having the api response, then I needed to upgrade it to an object. Maybe this would be better named data.

Reply:
I just renamed all the things to be more readable... in this case `metrics.metrics` becomes `metrics.data`.

Reply:
Ah okay, that helps a lot!

Reply:
Yeah, I also like that the tests needed to be updated, too.